packages/cra-template-typescript/template/src/modules/ui/components/Loader/Loader.tsx

```diff
@@ -19,7 +21,10 @@ const Loader = ({ show, children = null, inline = false, extend }: LoaderProps)
 
     return show ? (
         <div className={styles.loader}>
+            {/* // @use-antd-module-begin */}
             <AntdSpin />
+            {/* // @use-antd-module-end */}
+            Loading... {/* TODO delete this when antd-module is used....how? */}
```

Comment:
There's no way to use the negation of the `@use-antd-module` comment at this moment. So what about just having here the `Loading...` by default and having always removed the `AntdSpin`?

Reply:
Yes, that is the easiest solution, but this AntdSpin is the only component used in the template, thats the reason to keep it here.
I think its little bit better to keep both there and then delete the unwanted one

Reply:
I don't think this argument supports not removing it but whatever, just please remove the comment 🙏

Reply:
I meant like this is the only usage of Antd in the project so the user can easily see if everything works - but if you insist, i can delete it :)

Reply:
When #91 is ready - I will put a conditional render for that there

Reply:
I'll append this discussion to the issue actually, it'll be a good test case 😄